docs/software/admin.md

```diff
@@ -833,7 +835,24 @@ Notable field definitions
 * `badResponseNodes` : List of nodes that sent a malformed response
 * `topology` : Map of nodes to connection information
   * `inboundPeers`/`outboundPeers` : List of connection information by nodes
+    * `bytesRead`: The total number of bytes read from this peer.
+    * `bytesWritten`: The total number of bytes written to this peer.
+    * `duplicateFetchBytesRecv`: The number of bytes received that were duplicate transaction sets and quorum sets.
+    * `duplicateFetchMessageRecv`: The count of duplicate transaction sets and quorum sets received from this peer.
+    * `duplicateFloodBytesRecv`: The number of bytes received that were transactions and SCP votes duplicates.
+    * `duplicateFloodMessageRecv`: The count of duplicate transactions and SCP votes received from this peer.
+    * `messagesRead`: The total number of messages read from this peer.
+    * `messagesWritten`: The total number of messages written to this peer.
+    * `nodeId`: Node's public key.
+    * `secondsConnected`: The total number of seconds this node has been connected.
```

Comment:
Is this the number seconds the peer has been connected to this node specifically, or is it the number of seconds the peer has been connected to the network (the peer's uptime). I imagine it's the first, but it's ambiguous from the wording.

Reply:
Good point! It is the former. I've updated the wording to clarify this.